src/components/experiments/wizard/MinDiffCalculator.tsx

```diff
@@ -0,0 +1,378 @@
+import {
+  Button,
+  Checkbox,
+  InputAdornment,
+  Step,
+  StepContent,
+  StepLabel,
+  Stepper,
+  TextField,
+  Typography,
+} from '@material-ui/core'
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
+import { uniqueId } from 'lodash'
+import React, { useState } from 'react'
+
+import DebugOutput from 'src/components/general/DebugOutput'
+import { ExperimentFormData } from 'src/lib/form-data'
+import { isDebugMode } from 'src/utils/general'
+import {
+  defaultStatisticalPower,
+  defaultStatisticalSignificance,
+  samplesRequiredPerVariationForConversion,
+} from 'src/utils/math'
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {},
+    stepper: {
+      padding: 0,
+      background: 'none',
+    },
+    sideBySide: {
+      display: 'grid',
+      gridTemplateColumns: 'minmax(0, 1fr) minmax(0, 1fr)',
+      gridColumnGap: theme.spacing(2),
+    },
+    results: {
+      background: theme.palette.grey[100],
+      padding: theme.spacing(2),
+    },
+    adornment: {},
+    liability: {
+      display: 'flex',
+    },
+    liabilityCheckbox: {
+      alignSelf: 'start',
+    },
+  }),
+)
+
+function nanToZero(x: number): number {
+  return isNaN(x) ? 0 : x
+}
+
+function roundToNumPlaces(x: number, nPlaces: number): number {
+  return Math.round(x * Math.pow(10, nPlaces)) * Math.pow(10, -nPlaces)
```

Comment:
```suggestion
  return Math.round(x * Math.pow(10, nPlaces)) / Math.pow(10, nPlaces)
```

Because of JS precision issues with small numbers. 
Eg: `Math.pow(10, -4)` returns 0.0000999...

Reply:
Ah nice catch! I was wondering why I was getting some weirdness after rounding!